### DISCUSSION TITLE
When is the end of life of Prestashop 1.x ?

### DISCUSSION THREAD

**Discussion by citananda** [PrestaShop/PrestaShop#34254] [Q&A] [answered]
I would like to know if there is an end-of-life date for PrestaShop version 1.x.

**Comment by matks** [comment, marked as answer]
Hello!

https://build.prestashop-project.org/news/2023/178-in-extended-support-phase/

> This maintenance period [of 1.x] will end when PrestaShop 9.0.0 is released. When this day comes, PrestaShop 1.7 will not be maintained anymore.

**Reply by citananda** [reply]
Thanks @matks
